backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/PrototypeCleanupService.java

```diff
@@ -0,0 +1,112 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import de.bund.digitalservice.ris.norms.domain.entity.*;
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+/**
+ * Service responsible for cleaning up all {@link Dokument}s contained in a {@link Norm} since in the prototype
+ * not all available data can be published due to legal obligations
+ */
+@Service
+public class PrototypeCleanupService {
+
+  private static final Set<String> SHOULD_STAY = Set.of(
```

Comment:
i was confused for a short moment what exactly `SHOULD_STAY` contains and what it will be used for. Maybe move this down to the one method that is using it? or give it another name?

Reply:
I prefer personally to have them at the top of the class (like in the table of content service) but you can decide. 
About the naming what about "ALLOWED_METADATA" ?